GameData/RP-0/Tree/TREE-Engines.cfg

```diff
@@ -7696,6 +8096,46 @@ PART
 {
 }
 
+PARTUPGRADE
+{
+        name = RFUpgrade_GE4/J6H
+        partIcon = RO-H1-RS27 // FIXME Once we get dedicated model
+        techRequired = refinedTurbofans
+        entryCost = 0
+        cost = 0      
+        title = GE4 Engine Upgrade: GE4/J6H Config
+        basicInfo = Engine Performance Upgrade
+        manufacturer = Engine Upgrade
+        deleteme = 1
+        description = The GE4 Engine now supports the GE4/J6H configuration for increased performance. Unlock it in the VAB/SPH through the engine configs interface.\n\n
+}
+
+PART
+{
+    name = RFUpgrade_engineConfigSource_GE4/J6H
+    engineType = GE4
+}
+
+@PART[RFUpgrade_engineConfigSource_GE4/J6H]:AFTER[RealismOverhaulEngines]
+{
+    %MODULE[Module*EngineConfigs] {
+        @name = ModuleEngineConfigs
+        %CONFIG[GE4/J6H] {
+            &specLevel = operational
+        }
+    }
+}
+
+@PARTUPGRADE[RFUpgrade_GE4/J6H]:AFTER[RealismOverhaulEngines]
+{
+    %description = #$description$\nAvailable at specLevel $@PART[RFUpgrade_engineConfigSource_GE4/J6H]/MODULE[ModuleEngineConfigs]/CONFIG[GE4/J6H]/specLevel$
```

Comment:
Seeing a MM error here:
```
[ERR 09:12:04.380] Error - Cannot parse variable search when replacing (%) key description = #$description$
Available at specLevel $@PART[RFUpgrade_engineConfigSource_GE4/J6H]/MODULE[ModuleEngineConfigs]/CONFIG[GE4/J6H]/specLevel$ 
```
haven't dug into it, but my first guess is that the / in the config name isn't playing nice

Reply:
Removed /, everything appears to be working fine now.